packages/styles/scss/components/link-list/_link-list.scss

```
@@ -62,14 +62,16 @@
 
         .#{$prefix}--link-list__list--vertical-end & {
           border-bottom: 1px solid $ui-03;
-          max-width: 640px;
+          margin-left: -$carbon--spacing-05;
+          margin-right: -$carbon--spacing-05;
```

Thanks @caiobozato! According to the [visual specs](https://ibm.ent.box.com/file/662151736386), the `max-width` should be set to `640px`. Also, Carbon has a nifty mixin that will apply those negative margins for you 👍 

```suggestion
          max-width: 640px;
          
          @include hang;
```